src/routes/books.js

```diff
@@ -58,13 +57,21 @@ router.post('/', catchAsync(async (req, res) => {
 // Get a single book by ID.
 // TODO: Should render HTML, not return JSON.
 router.get('/:id', catchAsync(async (req, res) => {
+  let book;
+  let reviews;
+  let rating = 0;
   try {
-    const book = await Book.findByPk(req.params.id);
-    res.json(book);
+    book = await Book.findByPk(req.params.id);
+    reviews = await Review.findAll({where: {BookId: {[Op.eq]: book.id}}});
+    if(reviews.length > 0) {
+      rating = reviews.reduce((a, review) => a + review.Rating,0) / reviews.length;
```

Comment:
This code feels like something best placed in the `Book` or `Review` model instead, what do you think?